Lamina/include/Math/constants.hpp

```diff
@@ -0,0 +1,27 @@
+#ifndef LM_CONSTANTS
+#define LM_CONSTANTS
+
+#define PI			3.141592653589793238462643383279
+#define TAU			6.283185307179586476925286766559
+
+#define PI_FLOAT	(float)3.141592653589793238462643383279
```

Comment:
You can achieve the same thing by just adding an f at the end of the literal like this: `3.141592653589793238462643383279f`.